ui/js/page/settings/view.jsx

```diff
@@ -252,6 +278,49 @@ class SettingsPage extends React.PureComponent {
             </div>
           </div>
         </section>
+
+        <section className="card">
+          <div className="card__content">
+            <h3>{__("Key Fee")}</h3>
+          </div>
+          <div className="card__content">
+            <FormField
+              ref="max_key_fee"
+              type="number"
+              min="0"
+              placeholder="0"
+              step="0.01"
+              prefix={__("Max Key Fee")}
+              onChange={this.onKeyFeeChange.bind(this)}
+              defaultValue={daemonSettings.max_key_fee.amount}
+              disabled={daemonSettings.disable_max_key_fee}
+            />
+            {" "}
+            <FormField
+              type="select"
+              defaultValue={daemonSettings.max_key_fee.currency}
+              onChange={this.onFeeCurrencyChange.bind(this)}
+              disabled={daemonSettings.disable_max_key_fee}
+            >
+              <option value="USD">{__("US Dollars")}</option>
+              <option value="LBC">{__("LBRY credits")}</option>
+            </FormField>
+            <div className="form-field__helper">
+              {__(
+                "This value prevents you from downloading anything over this fee. (Default: 25 USD)"
```

Comment:
Change to: "This will prevent you from purchasing anything over this fee, as a safety measure."